src/wire/ipv6ext.rs

```diff
@@ -0,0 +1,356 @@
+use core::fmt;
+use {Error, Result};
+
+enum_with_unknown! {
+    /// IPv6 Extension Header Option Type
+    pub doc enum OptionType(u8) {
+        /// Pad1 option
+        Pad1 =  0,
+        /// PadN option
+        PadN =  1
+    }
+}
+
+impl fmt::Display for OptionType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            &OptionType::Pad1        => write!(f, "Pad1"),
+            &OptionType::PadN        => write!(f, "PadN"),
+            &OptionType::Unknown(id) => write!(f, "{}", id)
+        }
+    }
+}
+
+/// A read/write wrapper around a variable number of IPv6 Extension Header Options
```

Comment:
I don't think it is a variable number of IPv6 Extension Header Options. Each packet is only one Extension Header Option. E.g. if `bytes` contains the following

```
0x00, 0x01, 0x03, 0x00, 0x00, 0x00
```
This is a `Pad1` followed by a `PadN` with 3 bytes of padding, but when `packet = Packet::new(&bytes[..])` is run, `packet` is only the `Pad1` right?

Reply:
Yes, that is correct. Fixed.